tests/cloud_health_test.py

```diff
@@ -0,0 +1,156 @@
+    @patch('krux_cloud_health.cloud_health.CloudHealth')
+    def test_get_cloud_health_no_args(self, mock_cloud_health, mock_parser, mock_logger, mock_stats):
+        """
+        Cloud Health Test: All arguments created and passed into CloudHealth if none are provided.
+        """
+        mock_parser.return_value.parse_args.return_value = MagicMock(api_key=CloudHealthTest.API_KEY)
+        cloud_health = get_cloud_health()
+        mock_cloud_health.assert_called_once_with(
+            api_key = CloudHealthTest.API_KEY,
+            logger=mock_logger(name=NAME),
+            stats=mock_stats(prefix=NAME),
+            )
+
+    def test_cost_history_time_input(self):
+        """
+        Cloud Health Test: Cost history method properly passes in arguments to Get API call method.
+        with time_input provided
+        """
+        self.cloud_health._get_api_call = MagicMock()
+        cost_history = self.cloud_health.cost_history(Interval['daily'], 'time_input')
+        self.cloud_health._get_api_call.assert_called_once_with(
+            CloudHealthTest.COST_HISTORY_REPORT,
+            CloudHealthTest.API_KEY,
+            {'interval': 'daily', 'filters[]': 'time:select:time_input'},
+            )
+
+    def test_cost_history_no_time_input(self):
+        """
+        Cloud Health Test: Cost history method properly passes in arguments to Get API call method.
+        without a specified time_input
+        """
+        self.cloud_health._get_api_call = MagicMock()
+        cost_history = self.cloud_health.cost_history(Interval['daily'])
+        self.cloud_health._get_api_call.assert_called_once_with(
+            CloudHealthTest.COST_HISTORY_REPORT,
+            CloudHealthTest.API_KEY,
+            {'interval': 'daily'},
+            )
+
+    def test_cost_current(self):
+        """
+        Cloud Health Test: Cost current method properly passes in arguments to Get API call method.
+        """
+        self.cloud_health._get_api_call = MagicMock()
+        cost_current = self.cloud_health.cost_current()
+        self.cloud_health._get_api_call.assert_called_once_with(
+            CloudHealthTest.COST_CURRENT_REPORT,
+            CloudHealthTest.API_KEY
+            )
+
+    @patch('krux_cloud_health.cloud_health.pprint.pformat')
+    @patch('krux_cloud_health.cloud_health.requests')
+    def test_get_api_call(self, mock_request, mock_pprint):
+        """
+        Cloud Health Test: Get API call method calls API with valid report and API key.
+        """
+        api_call = {'api_call': 'return'}
+        mock_request.get.return_value.json.return_value = api_call
+        get_api_call = self.cloud_health._get_api_call(CloudHealthTest.COST_HISTORY_REPORT, CloudHealthTest.API_KEY)
+        mock_pprint.assert_called_once_with(api_call)
+        self.cloud_health.logger.debug.assert_called_once_with(mock_pprint(api_call))
+
+    @patch('krux_cloud_health.cloud_health.requests')
+    def test_get_api_call_error(self, mock_request):
```

Comment:
Please test the message of the exception thrown is same as the error value from API. (i.e. If `api_call == {'error': 'foo'}`, the message of `ValueError` should be `'foo'`.)